reactor-netty-core/src/test/java/reactor/netty/tcp/TcpClientTests.java

```diff
@@ -1372,4 +1374,37 @@ private void doTestSharedNameResolver(TcpClient client, boolean sharedClient) th
 			    .block();
 		}
 	}
+
+	/* https://github.com/reactor/reactor-netty/issues/1765 */
+	@Test
+	void noSystemProxySettings() {
+		Properties props = System.getProperties();
+		assumeThat(!(props.containsKey("http.proxyHost") || props.containsKey("https.proxyHost")
```

Comment:
for tracking purpose, it is a bit of a stretch but having an `assumeThat` here could hide issues in the future if a regression is introduced to `proxyWithSystemProperties()`-without-actual-properties AND the test environment does contain a proxy configuration.